### PR TITLE
Document GHC-30606

### PR DESCRIPTION
**Pull request by isomorpheme**

### CODE REVIEW

File: message-index/messages/GHC-30606/example-equality/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Explanation
+
+This example declares `f` with the constraints `(a ~ b, b ~ a)` (`a` is
+equal to `b` and `b` is equal to `a`), but `a ~ b` already implies
+that `b ~ a` (in other words, `~` is a [symmetric relation]).
+
+We fix this by removing the constraint which is marked as redundant in the error
+message, namely `b ~ a`.
+
+[symmetric relation]: https://en.wikipedia.org/wiki/Symmetric_relation
```
Comment (InverseIntegral) [inline message-index/messages/GHC-30606/example-equality/index.md:26]: I guess this should be on line 21 as a link?

**Review by InverseIntegral — CHANGES_REQUESTED** [review]

File: message-index/messages/GHC-30606/example-equality/index.md (hunk 1 above)
Comment (david-christiansen) [reply message-index/messages/GHC-30606/example-equality/index.md:26]: That syntax works here when I render the site - I think it's fine

Comment (david-christiansen) [reply message-index/messages/GHC-30606/example-equality/index.md:26]: Does it not work for you?

Comment (InverseIntegral) [reply message-index/messages/GHC-30606/example-equality/index.md:26]: Oh yes you are right. My bad :sweat_smile:

**Review by InverseIntegral — APPROVED** [review]

**Pull request merged**